background.js

```diff
@@ -28,6 +28,20 @@ function getAdStatus(json) {
     }
 }
 
+const adHideScript = shouldHide => `
+    const video = document.querySelector(".video-ref.Layout-sc-1xcs6mc-0 > video");
```

Comment:
@kwyntes Please rename this const variable. 

![image](https://github.com/user-attachments/assets/f83014dd-2d52-4119-bd9f-f173398531b8)

Scope the name to the plugin to something like `hideableVideo` or along those lines, so that it's not conflicting with other variables already present in the global scope of the tab.

Additionally, please check to make sure that the video player is correctly displayed again after ads stop. When I tested, the video player did not correctly show again. The player was still hidden and the helper text was still present. It might be possible that this was due to the naming conflict error.

Here are instructions on how to temporarily load the updated script into Firefox locally for testing.

https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/

Reply:
Ah I see what the problem is. It's not that the name `video` is already used by Twitch, it's that it's used by this extension twice, since I'm redeclaring the `const video` by injecting the script a second time, which isn't actually needed as `video` already points to the right thing. We can simply make the declaration of `const video` conditional on `shouldHide` and then it'll work.